src/components/Form/ProfileForm.tsx

```diff
@@ -75,30 +77,38 @@ function ProfileForm({ callback }: { callback?: () => void }) {
   ) => {
     if (user && walletClient && publicClient) {
       try {
-        const cid = await postToIPFS(
-          JSON.stringify({
-            title: values.title,
-            role: values.role,
-            image_url: values.image_url,
-            video_url: values.video_url,
-            name: values.name,
-            about: values.about,
-            skills: values.skills,
-            web3mailPreferences: user.description?.web3mailPreferences,
-          }),
-        );
+        let cid = await tlClient?.profile.upload({
+          title: values.title,
+          role: values.role,
+          image_url: values.image_url,
+          video_url: values.video_url,
+          name: values.name,
+          about: values.about,
+          skills: values.skills,
```

Comment:
check the diff, there is a regression for web3mailPreferences: user.description?.web3mailPreferences,

Reply:
globally to prevent duplicate of this object, create an intermediaries variable

Reply:
To fix this, i need to create a fix in the sdk as well.